components/post_markdown/index.js

```diff
@@ -10,6 +10,7 @@ function mapStateToProps(state, ownProps) {
     return {
         channel: getChannel(state, ownProps.channelId),
         pluginHooks: state.plugins.components.MessageWillFormat,
+        codeBlockPlugins: state.plugins.components.CodeBlockComponent || [],
```

Comment:
nit: would it make sense to continue to use the term `codeBlockComponent`?

Reply:
Done :+1: